lib/commands/get.js

```diff
@@ -45,21 +45,19 @@ module.exports = {
     let TabSelector = require('../tasks/select-tab.js');
     let tabSelector = new TabSelector({ project: this.project, tabName: options.tab });
 
-    return tabSelector.run().then(tab => this._rows(tab));
+    return tabSelector.run().then(tabs => {
+      return Promise.all(tabs.map(this._rows.bind(this))).then(rowsByTab => {
+        let mergedRows = [].concat.apply([], rowsByTab);
```

Comment:
I'm concatenating the result of all the promises, each result contains an array with the rows of a tab. I'm using the apply method to flatten the array.

For better understanding see the response here http://stackoverflow.com/questions/10865025/merge-flatten-an-array-of-arrays-in-javascript